### DISCUSSION TITLE
[resolved] v5 Problem with join and where

### DISCUSSION THREAD

**Discussion by brandonsousa** [adonisjs/core#3272] [Help]
I'm trying to join where but when the where clause is passed the code doesn't bring any results.

`await Database.from('TABLE_A')
			.select('TABLE_A.name')
			.join('TABLE_B', 'TABLE_B.id', '=', 'TABLE_A.TABLE_B_id')
			.join('TABLE_C', 'TABLE_C.id', '=', 'TABLE_B.TABLE_C_id')
			.join('TABLE_D', 'TABLE_D.id', '=', 'TABLE_C.TABLE_D_id')
			.join('TABLE_E', 'TABLE_E.id', '=', 'TABLE_A.TABLE_E_id')
			.join('TABLE_F', 'TABLED_D.id', '=', 'TABLE_E.TABLE_F_id')
			.where('TABLE_F.id', '=', id)
			.orderBy('TABLE_A.created_at', 'asc')
			.paginate(1, 10);`

**Comment by brandonsousa** [comment]
added `as` on inner join and where clause

**Comment by mybigman** [comment]
Why does your join not join to `table_f` ?

```js
.join('TABLE_F', 'TABLED_D.id', '=', 'TABLE_E.TABLE_F_id')
```
